README.md

```diff
@@ -58,15 +59,25 @@ System.Console.WriteLine("Session expires at: {0}", session.ExpireTime);
 It is recommended to store the auth token from the session and check at startup if it has expired. If the token has expired you must reauthenticate. The expiry time of the token can be changed as a setting in the server.
 
 ```csharp
-var authtoken = "restored from somewhere";
-var session = Session.Restore(authtoken);
-if (session.IsExpired)
+var authToken = "restored from somewhere";
```

Comment:
@lugehorsam It's too late to rename the field in the protocol because it would completely break backwards compatibility. I don't think it should be renamed because the field is also called `ISession.AuthToken`.

Perhaps it would have been better to have named it token though that would be more confusing now because we have both session and refresh tokens but I think the opportunity for that rename is long gone.